egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -81,11 +81,6 @@ def get_args():
     parser.add_argument("--chain.leaky-hmm-coefficient", type=float,
                         dest='leaky_hmm_coefficient', default=0.00001,
                         help="")
-    parser.add_argument("--chain.apply-deriv-weights", type=str,
```

Comment:
You'll have to leave this option present and just say that it's deprecated in the usage message 
and do nothing with it.    Otherwise, if people have locally modified scripts with this option, they
will crash.